modules/installation-guide/partials/proc_creating-an-instance-of-the-che-operator.adoc

```diff
@@ -1,44 +1,60 @@
-//This module is included in
+//This module is included in:
 //
-// * assembly_installing-che-using-the-che-operator-in-openshift-4-web-console
+// assembly_installing-che-on-openshift-4-using-operatorhub
 
 [id="creating-an-instance-of-the-{prod-id-short}-operator_{context}"]
 = Creating an instance of the {prod} Operator
 
-Follow this procedure to install {prod} with the default configuration. To modify the configuration, see xref:configuring-the-che-installation.adoc[].
+Follow this procedure to install {prod-short} with the default configuration. To modify the configuration, see xref:configuring-the-che-installation.adoc[].
 
 .Procedure
 
-. To create an instance of the {prod} Operator, in the left panel, navigate to the *Operators* -> *Installed Operators* section.
+<<<<<<< HEAD
+. Using the left panel, navigate to the *Operators* -> *Installed Operators* page.
 
-. In the *Installed Operators* screen, click the *{prod}* name.
+. In the *Installed Operators* page, click the *{prod}* name.
 
-. In the *Operator Details* screen, in the *Details* tab, inside of the *Provided APIs* section, click the *Create Instance* link.
+. In the *Operator details* page, in the *Details* tab, click the *Create instance* link in the *Provided APIs* section.
++
+A user is navigated to the *Create CheCluster* page, which contains the configuration stored in the `CheCluster` Custom Resource, needed to create a {prod-short} instance. 
```

Comment:
-> The user is navigated (?)
...but this is honestly still a very awkward construction, can you perhaps just use "This navigates you..."?